tests/end2end/playwright/filter-layer-by-user.spec.js

```diff
@@ -12,17 +12,34 @@ test.describe('Filter layer data by user - not connected', () => {
         await page.locator('#dock-close').click();
     });
 
-    test('GetMap', async ({ page }) => {
-        // Hide all elements but #newOlMap and its children
-        await page.$eval("*", el => el.style.visibility = 'hidden');
-        await page.$eval("#newOlMap, #newOlMap *", el => el.style.visibility = 'visible');
+    // DISABLED BECAUSE IT IS NOT RELIABLE, MAINTAINABLE AND CAUSES HEADACHE ;-)
+    // Instead, we use a WMS GetFeatureInfo in JSON format below
+    // test('GetMap', async ({ page }) => {
+    //     // Hide all elements but #newOlMap and its children
+    //     await page.$eval("*", el => el.style.visibility = 'hidden');
+    //     await page.$eval("#newOlMap, #newOlMap *", el => el.style.visibility = 'visible');
+
+    //     expect(await page.locator('#newOlMap').screenshot()).toMatchSnapshot('map_not_connected.png', {
+    //         maxDiffPixels: 500
+    //     });
+    // });
+
+    test('WMS GetFeatureInfo JSON', async ({ page }) => {
+
+        const getFeatureInfo = await page.evaluate(async () => {
+            return await fetch("/index.php/lizmap/service?repository=testsrepository&project=filter_layer_by_user&SERVICE=WMS&REQUEST=GetFeatureInfo&VERSION=1.3.0&CRS=EPSG%3A2154&INFO_FORMAT=application%2Fjson&QUERY_LAYERS=green_filter_layer_by_user_edition_only%2Cblue_filter_layer_by_user%2Cred_layer_with_no_filter&LAYERS=green_filter_layer_by_user_edition_only%2Cblue_filter_layer_by_user%2Cred_layer_with_no_filter&STYLE=default%2Cdefault%2Cdefault&BBOX=-310600.3821764754%2C726545.2026738503%2C364617.6349262256%2C1244071.2377259205&FEATURE_COUNT=10&FILTER=green_filter_layer_by_user_edition_only:\"gid\" > 0")
```

Comment:
```suggestion
            return await fetch("/index.php/lizmap/service?repository=testsrepository&project=filter_layer_by_user&SERVICE=WMS&REQUEST=GetFeatureInfo&VERSION=1.3.0&CRS=EPSG%3A2154&INFO_FORMAT=application%2Fjson&QUERY_LAYERS=green_filter_layer_by_user_edition_only%2Cblue_filter_layer_by_user%2Cred_layer_with_no_filter&LAYERS=green_filter_layer_by_user_edition_only%2Cblue_filter_layer_by_user%2Cred_layer_with_no_filter&STYLE=default%2Cdefault%2Cdefault&BBOX=-310600%2C726545%2C364617%2C1244071&FEATURE_COUNT=10&FILTER=green_filter_layer_by_user_edition_only:\"gid\" > 0")
```

It's already a very long line, we don't need to go below meters :) (we can split it later)